### DISCUSSION TITLE
Corrections in achievments section

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#143] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/162660470-6d3f1151-a09e-4517-a8a8-ab83ae6572d9.png)

@gimyboya 
Zakat Beneficials   ->Zakat Beneficiaries

Orphans & Widows Supported   -> Orphans & Widows Supported

Well Built -> Wells built

I need to get updated statistics

**Comment by an-nadaa** [comment]
This should be fixed
